src/main/resources/templates/emails.html

```diff
@@ -0,0 +1,70 @@
+        -webkit-background-size: cover;
+        -moz-background-size: cover;
+        -o-background-size: cover;
+        background-size: cover;
+    }
+    * {
+        margin: 0;
+        padding: 0;
+    }
+    h1,h2 { text-align: center }
+    html,
+    body {
+        height: 100%;
+        display:flex;
+        flex-direction:column;
+    }
+    footer{
+        margin-top:auto;
+        text-align:center;
+    }
+</style>
+<body>
+        <h2>Emails</h2>
+        <table>
+            <thead>
+            <tr>
+                <th>Id of email</th>
+                <th>Recipient</th>
+                <th>Subject</th>
+                <th>Main text of massage</th>
+                <th>Send date</th>
+            </tr>
+            </thead>
+            <tbody>
+            <tr th:each="emails : ${emails}">
```

Comment:
I recommend to specify in comments on top of this Thymeleaf fragment type of variables that are used here.
Something like this:
`emails          - java.util.List<com.paskar.model.Email>`